### ISSUE TITLE
Auto facettes, auto tags

### ISSUE THREAD

**Issue by mastersign** [mastersign/dashops#9]
Generate facette values and tags automatically when certain conditions are met.

**Examples:**

* If facette `host` matches regex `-production$` set facette `environment` to `production`
* If facette `verb` is `install` add tag `setup`

**Comment by mastersign** [comment]
Draft for configuration format:

```yaml
auto:
  - match:
      mode: facette
      facette: host
      regex: '-production$'
    facettes:
      environment: production
  - match:
      mode: facette
      facette: verb
      value: install
    tags: ['setup']
```

**Comment by mastersign** [comment]
Counter proposal supporting multiple include and exclude patterns:

```yaml
auto:
  - include:
      - mode: facette
        facette: host
        regex: '-production$'
    facettes:
      environment: production
  - include:
      - mode: facette
        facette: verb
        value: install
    tags: ['setup']
```

**Comment by mastersign** [comment]
implemented in 421b248a4d25b94838ccf982bd3309b427525334
